src/native/libs/System.IO.Ports.Native/pal_termios.c

```diff
@@ -437,22 +437,21 @@ int32_t SystemIoPortsNative_TermiosReset(intptr_t handle, int32_t speed, int32_t
 
     if (tcgetattr(fd, &term) < 0)
     {
-        return  -1;
+        return -1;
     }
 
 #if HAVE_CFMAKERAW
     cfmakeraw(&term);
 #else
-    term.c_iflag &= ~(IMAXBEL|IGNBRK|BRKINT|PARMRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON);
+    term.c_iflag &= ~(IMAXBEL | IGNBRK | BRKINT | PARMRK | ISTRIP | INLCR | IGNCR | ICRNL | IXON);
```

Comment:
@janvorli, it is set here. The redundant line 455, which was overwriting all the flags, was deleted.